### PR TITLE
feat: add casa cases section without court dates to admin home page

### LINKED ISSUE #4224: Add a Section to the Admin Cases Page for Cases Without Court Dates
**What type(s) of user does this feature affect?**
 - admins

**Description**
Add a section to the admin homepage that displays cases that don't have any court dates listed.

**Screenshots of current behavior, if any**  
You can paste images on the clipboard here

_QA Login Details:_  
[Link to QA site](https://casa-qa.herokuapp.com/)  

Login Emails: 
- volunteer1@example.com  view site as a volunteer
- supervisor1@example.com view site as a supervisor
- casa_admin1@example.com view site as an admin

password for all users: 12345678  

### Questions? Join Slack!

We highly recommend that you join us in slack https://rubyforgood.herokuapp.com/ #casa channel to ask questions quickly and hear about office hours (currently Tuesday 6-8pm Pacific), stakeholder news, and upcoming new issues.

### PR DESCRIPTION
**Pull request by tonyaraujop**
### What github issue is this PR for, if any?
Resolves #4224 

### What changed, and why?
Add section to admin's home page to list casa cases without court dates

### How will this affect user permissions?
- Volunteer permissions:
- Supervisor permissions:
- Admin permissions:

### How is this tested? (please write tests!) 💖💪
- Tests for specific given contexts, using casa cases with/without court dates

### Screenshots please :)
- image

### Feelings gif (optional)
What gif best describes your feeling working on this issue? https://giphy.com/
How to embed:
`alt text`

### Feedback please? (optional)
We are very interested in your feedback! Please give us some :) https://forms.gle/1D5ACNgTs2u9gSdh9

### CODE REVIEW

File: app/models/casa_case.rb [hunk 1]
```diff
@@ -97,6 +97,10 @@ class CasaCase < ApplicationRecord
     where(active: false)
   }
 
+  scope :missing_court_dates, -> {
+    where.missing(:court_dates)
```
Comment (littleforest) [inline app/models/casa_case.rb:101]: cool, I did not know about `missing` !

File: spec/requests/supervisors_spec.rb [hunk 2]
```diff
@@ -16,6 +16,44 @@
   web_mock = WebMockHelper.new(blacklist)
   web_mock.stub_network_connection
 
+  describe "GET /index" do
+    let!(:active_supervisor) { create(:supervisor, casa_org: org, active: true) }
+    let!(:inactive_supervisor) { create(:supervisor, casa_org: org, active: false) }
+
+    it "returns http status ok" do
+      sign_in admin
```
Comment (littleforest) [inline spec/requests/supervisors_spec.rb:24]: Would you mind just defining `active_supervisor` and `inactive_supervisor` within the `it` block and avoiding the use of `let` (and modifying specs below accordingly)? It's easier to follow the specs, especially for new developers.

https://thoughtbot.com/blog/lets-not

**Review by littleforest — COMMENTED** [review]
Looking good! Can you fix merge conflicts?

File: spec/requests/supervisors_spec.rb (hunk 2 above)
Comment (tonyaraujop) [reply spec/requests/supervisors_spec.rb:24]: yeah sure, I forgot to move this after I rebase the last time. Thanks for noticing this.

**Comment by tonyaraujop** [comment]
Hey @littleforest I think is good to review now. I'm seeing 2 failing specs but they're passing passing locally. Are they flaky tests?

**Review by littleforest — APPROVED** [review]
Thanks for adding specs!

**Pull request merged**